list_resources.sh

```diff
@@ -1,19 +1,23 @@
-#!/bin/bash
-specs_dir="./specs"
+#!/bin/sh
 
-function findResourceDir(){
-    for file in `ls $1`
-    do
-    	work_dir=$1"/"$file
-        if [ -d $work_dir ]
-        then
-        	if [ -n "$(git status -s $work_dir)" ] && [ -f $work_dir"/api.yaml" ] \
-        		&& ([ -f $work_dir"/ansible.yaml" ] || [ -f $work_dir"/terraform.yaml" ]); then
-				echo ${work_dir:8}		# remove the directory prefix './specs/'
+findResourceDir(){
+	for file in `ls $1`
+	do
+		work_dir=$1"/"$file
+		if [ -d $work_dir ]
+		then
+			if [ -n "$(git status -s $work_dir)" ] && [ -f $work_dir"/api.yaml" ] \
```

Comment:
We don't need to check existence of YAML files, MM can handle it very well.

To make the code logic simple, please just get a list of changed/added folder names, that's all.

Reply:
Removed.